README.md

```diff
@@ -1,111 +1,63 @@
+We also have a [forum][Discourse] and a [Slack][Slack].
 
-We hope you enjoy SuperCollider; please get involved in the active community!
+[list signup]: http://www.birmingham.ac.uk/facilities/ea-studios/research/supercollider/mailinglist.aspx
+[sc-users archives]: https://www.listarc.bham.ac.uk/lists/sc-users/
+[sc-dev archives]: https://www.listarc.bham.ac.uk/lists/sc-dev/
+[Discourse]: https://scsynth.org/
+[Slack]: https://join.slack.com/t/scsynth/shared_invite/enQtMzk3OTY3MzE0MTAyLWY1ZGE1MTJjYmI5NTRkZjFmNjZmNmYxOWI0NDZkNjdkMzdkNjgxNTJhZGVlOTEwYjdjMDY5OWM0ZTA4NWFiOGY
 
+Contribute
+----------
 
-Reporting bugs
---------------
+Development of SuperCollider happens here on GitHub, and we are grateful to the community for contributing bugfixes and improvements. Read below to learn how you can take part in improving SuperCollider.
 
-Please report bugs to the GitHub issue tracker or discuss on the sc-users mailing list.
+### [Code of Conduct](CODE_OF_CONDUCT.md)
 
-https://github.com/supercollider/supercollider/issues
+Please read our adopted [code of conduct](CODE_OF_CONDUCT.md) before contributing, so that you can understand what actions will and will not be tolerated.
 
+### [Contributing Guide](CONTRIBUTING.md)
 
-Building the Source Code
-------------------------
+Read our [contributing guide](CONTRIBUTING.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes.
 
-Platform specific build and install instructions can be found in the following files:
 
- - [README_MACOS.md](README_MACOS.md)
- - [README_LINUX.md](README_LINUX.md)
- - [README_WINDOWS.md](README_WINDOWS.md)
- - [README_IPHONE.md](README_IPHONE.md)
- - [README_JAILBROKEN_IPHONE.md](README_JAILBROKEN_IPHONE.md)
+### Good First Issues
 
+To help you get your feet wet and get you familiar with our contribution process, we have a list of [good first issues](https://github.com/supercollider/supercollider/labels/good%20first%20issue) that contain bugs which have a relatively limited scope. This is a great place to get started. You can also ask on our developer's mailing list or on [Slack][Slack].
 
 License
 -------
 
-SuperCollider is free software; you can redistribute it and/or modify it under
-the terms of the GNU General Public License as published by the Free Software
-Foundation; either version 3 of the License or (at your option) any later
-version. See [COPYING](COPYING) for the license text.
-
-SuperCollider was originally published under the terms of version 2 or later of the GPL, but the
-application now includes some GPL3-licensed code. If you have a need specifically for GPL2
-compatibility then it is possible to recompile SuperCollider from source without the GPL3 elements.
-To do this, use the CMake option `-DNO_GPL3=ON`.
-
-SuperCollider's help documentation is published under a Creative Commons license.
-
-- SuperCollider 3.1 was released on 2007-10-31
```

Comment:
we talked about where this history should go, right? AFAICT there's no place where all this version history is actually kept, so idk how I feel about removing it outright. What are your thoughts on keeping a brief history section?

Reply:
why there?

Reply:
actually -- how about right at the top of the changelog?

Reply:
Yes!